src/elements/content-preview/ContentPreview.js

```diff
@@ -289,10 +295,34 @@ class ContentPreview extends React.PureComponent<Props, State> {
             // eslint-disable-next-line react/no-unused-state
             prevFileIdProp: fileId,
         };
+
         const { logger } = props;
         logger.onReadyMetric({
             endMarkName: MARK_NAME_JS_READY,
         });
+
+        if (this.props.useV4BoxAnnotations && !this.props.boxAnnotations && !global.BoxAnnotations) {
+            importV4Annotations().then(() => {
+                const features = {
+                    discoverability: true,
+                    drawing: true,
+                };
```

Comment:
I am uncertain if I should be setting this to true for the consumer or if the consumer should have control of this